contributors/devel/sig-scheduling/scheduling_code_hierarchy_overview.md

```diff
@@ -0,0 +1,280 @@
+# Scheduler code hierarchy overview
+
+## Introduction
+
+The scheduler watches for newly created Pods that have no Node assigned.
+For every Pod that the scheduler discovers, the scheduler becomes responsible
+for finding the best Node for that Pod to run on.
+Scheduling in general is quite an extensive field in computer science which takes
+into account various range of constraints and limitations.
+Each workload may require a different approach to achieve optimal scheduling results.
+The kube-scheduler provided by Kubernetes project was constructed with a goal
+to provide high throughput at the cost of being simple.
+To help in building a scheduler (the default or a custom one) and to share
+elements of the scheduling logic,
+[the scheduling framework](https://www.google.com/url?q=https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/&sa=D&source=editors&ust=1617027676318000&usg=AOvVaw2vEsCZwet-BiQGhy0mVpZu)
```

Comment:
Simplify the URL:  https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/